tsl/test/CMakeLists.txt

```diff
@@ -59,7 +59,22 @@ if(PG_REGRESS)
     ${PG_REGRESS_OPTS_LOCAL_INSTANCE}
     USES_TERMINAL)
 
+  add_custom_target(provecheck
+    rm -rf ${CMAKE_CURRENT_BINARY_DIR}/tmp_check &&
+    PERL5LIB="$ENV{HOME}/perl5/lib/perl5:$ENV{PERL5LIB}"
+    BUILDIR=${CMAKE_BINARY_DIR}
+    TESTDIR=${CMAKE_CURRENT_BINARY_DIR}
+    PATH="${PG_BINDIR}:$ENV{PATH}"
+    PGPORT=6${TEST_PGPORT_LOCAL}
+    PG_REGRESS=${PG_REGRESS}
+    REGRESS_SHLIB="${PG_SOURCE_DIR}/src/test/regress/regress.so"
```

Comment:
This is tied to Linux, so this means that we cannot run it on Windows.

Reply:
I removed `REGRESS_SHLIB` and the commands still work ok. So no linux dependency as of now.